fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -2053,5 +2053,11 @@ public class Config extends ConfigBase {
 
     @ConfField(mutable = true)
     public static boolean use_mysql_bigint_for_largeint = false;
+
+    /** 
+    * the max package size fe thrift server can receive,avoid accepting error or too large package causing OOM,default 20M
+    */
+    @ConfField
+    public static int fe_thrift_max_pkg_size = 20000000;
```

Comment:
rename to fe_thrift_max_pkg_bytes

Reply:
I have renamed fe_thrift_max_pkg_size to fe_thrift_max_pkg_bytes